frontend/src/types.ts

```diff
@@ -474,7 +492,7 @@ export type PathType = typeof PAGEVIEW | typeof AUTOCAPTURE | typeof SCREEN | ty
 export type RetentionType = 'retention_recurring' | 'retention_first_time'
 
 export interface FilterType {
-    insight: InsightType
+    insight?: InsightType
```

Comment:
Not sure about this change, shouldn't insight be present all the time?

Reply:
When we go from `LocalFilter[]` to `FilterType` the insight is unknown.